### PR TITLE
Add sonarqube task to project configuration

### PR DESCRIPTION
**Pull request by lgonczar**
### Change description ###

Add sonarqube task to project configuration (required by the pipeline)

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -89,6 +90,18 @@ jacocoTestReport {
...
 
+project.tasks['sonarqube'].dependsOn test, integration
+
+sonarqube {
+  properties {
+    property "sonar.projectName", "Reform :: Bulk Scan Orchestrator"
+    property "sonar.jacoco.reportPath", "${project.buildDir}/jacoco/test.exec"
+
+    // TODO: uncomment the following line when integration tests are created
+    // property "sonar.jacoco.itReportPath", "${project.buildDir}/jacoco/integration.exec"
```
Comment (rkondratowicz) [inline build.gradle]: it may be safer to add an empty integration package (or with dummy test) ;)

Comment (lgonczar) [reply build.gradle]: I've started considering that, too. Will do.

**Comment by doncem** [comment]
Perhaps worth also adding sonarcloud step similarly like we have in processor repo? I think token is same across organisation so https://github.com/hmcts/bulk-scan-processor/blob/master/.travis.yml#L3-L7 and other lines in travis config could be just copied.

**Comment by lgonczar** [comment]
I'll add it for consistency with bulk-scan-processor

**Comment by lgonczar** [comment]
Not adding it in the end - we're waiting to be added to hmcts organisation in sonarcloud

File: build.gradle [hunk 2]
```diff
@@ -89,6 +90,16 @@ jacocoTestReport {
   }
 }
 
+project.tasks['sonarqube'].dependsOn test, integration
+
+sonarqube {
+  properties {
+    property "sonar.projectName", "Reform :: Bulk Scan Orchestrator"
```
Comment (rkondratowicz) [inline build.gradle]: i'm not sure, but `settings.gradle` file may also be needed (check other repos)

**Pull request merged**